src/components/Claim/Claim.tsx

```
@@ -1,28 +1,24 @@
-import * as utils from '@/utils'
-import * as context from '@/context'
 import * as assets from '@/assets'
 import * as consts from '@/constants'
 import * as _utils from './Claim.utils'
 import FlowManager from './Link/FlowManager'
-import { ActionType, estimatePoints } from '../utils/utils'
+import { useSuspenseQuery } from '@tanstack/react-query'
+import { Attachement, CheckLinkReturnType } from './types'
+import { getClaimQuery } from './services/query'
 
 export const Claim = ({}) => {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)